.gitignore

```
@@ -0,0 +1,2 @@
+private-values.yaml
```

Curious to hear others thoughts on if we should have this ignore all private pre-fixed items? 

I believe it is:
```suggestion
private-*
```

---

I can't think of anything we would store in git that starts with `private..` but generally speaking it is better to be precise with this vs. blanket, in case someone a year from now has a legitimate `private...` item and gets blocked 

fwiw, I know that `*~` for emacs backup files is a well know gitignore item (and one I ran into today ;-) )

---

wdyt @johnmcollier @coreydaley @thepetk ? ^^^

---

I'm good with keeping it precise!

---

If we are going to include a glob of files, i would rather exclude a directory that can be used instead, otherwise I think the more precise one is better.

---

Yeah, I'm fine with just `private-values.yaml` for now.